recipes/eva-sub-cli/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+VCF_VALIDATOR_VERSION=0.9.6
+BIOVALIDATOR_VERSION=2.1.0
+EVA_PYUTILS_VERSION=0.6.1
+
+EVA_SUB_CLI="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}"
+mkdir -p ${PREFIX}/bin ${EVA_SUB_CLI}
+
+# Install eva-sub-cli
+$PYTHON -m pip install .
+cp bin/* ${PREFIX}/bin
+echo "Done with eva-sub-cli"
+
+cd ${EVA_SUB_CLI}
+
+# Install python dependencies not yet on conda
+curl -Lo eva-pyutils.zip https://github.com/EBIvariation/eva-common-pyutils/archive/refs/tags/v${EVA_PYUTILS_VERSION}.zip \
```

Comment:
We should not do that, but add those dependencies to conda.

Reply:
We'll work on doing this :+1:

Reply:
now added